### ISSUE TITLE
Change in proof triggers when it shouldn't

### ISSUE THREAD

**Issue by pbelmans** [stacks/stacks-history#1]
Looking at https://stacks.math.columbia.edu/tag/0200/history we see that `stacks-history` incorrectly thinks that in https://github.com/stacks/stacks-project/commit/18e0cef1a4325a51cd422490304cb5ac4d1acf04/coherent.tex#diff-05da2b9356b5504b86d4c56002638537R4547 both the statement and the proof were changed. It's not just my output code which does something bad, it's really in the database this way, which means it came from `stacks-history` in this way, somehow.

Unless of course the mistake is in the part of the update script that recognises what has changed. But as I want to integrate this code into `stacks-history` in any case, I'm putting the issue here.

**Comment by aisejohan** [comment]
Dear Pieter, I checked and the script `stacks-history/history.py` correctly finds only a change in the statement of the lemma and only that a reference got added. I sort of think this is the right thing for this script to do.

Maybe when we are importing the history into the database (of the new website) we can detect that this change is "adding a reference"?

About the current website: I fiddled with the logic in `stacks-website/php/pages/taghistory.php` twice and this may be the cause for it saying "both statement and proof got changed". The commits were 2e18175fdca6ba462d1ec4c9f35cb6de2365c6a5 and 97dc8078c309d3457f46899d689264733113962e
